packages/perseus/src/styles/perseus-renderer.less

```diff
@@ -252,7 +252,10 @@
     }
 
     .svg-image {
-        display: inline-block;
+        display: block;
+        margin: 0 auto;
+        margin-inline-start: auto;
+        margin-inline-end: auto;
```

Comment:
Do we need this if this image is centered? If so, could we add a comment regarding what it's for?